gammapy/cube/fit.py

```diff
@@ -346,6 +346,49 @@ def create(
             gti=gti,
         )
 
+    def stack(self, other):
+        """
+        Stack the maps in other onto self.
+        The stacking is done inplace
+
+        Parameters
+        ----------
+        other: `~gammapy.cube.MapDataset`
+            the MapDatasets to be stacked with this one.
+
+        Returns
```

Comment:
As the stacking is done in place, nothing is returned. So just remove this section...